web/src/web/stores.py

```diff
@@ -26,15 +24,16 @@
 
 
 def _get_or_refresh_cache(
-    task: dict, url: str = None, expires: int = None
+    task: str, url: Optional[str] = None, expires: Optional[int] = None
```

Comment:
@docsteveharris I think this needs to be a string here rather than a dict, as we're indexing into another dictionary with this object - can you confirm?